ddtrace/contrib/pytest/plugin.py

```diff
@@ -756,3 +753,42 @@ def pytest_runtest_makereport(item, call):
             span.set_tag_str(test.RESULT, test.Status.XPASS.value)
         if call.excinfo:
             span.set_exc_info(call.excinfo.type, call.excinfo.value, call.excinfo.tb)
+
+
+@pytest.hookimpl
+def pytest_addhooks(pluginmanager):
+    from ddtrace.contrib.pytest import newhooks
+
+    pluginmanager.add_hookspecs(newhooks)
+
+
+@pytest.hookimpl(trylast=True)
+def pytest_ddtrace_get_item_module_name(item):
+    pytest_module_item = _find_pytest_item(item, pytest.Module)
+    pytest_package_item = _find_pytest_item(pytest_module_item, pytest.Package)
+
+    if _module_is_package(pytest_package_item, pytest_module_item):
+        return pytest_package_item.module.__name__
+    return pytest_module_item.nodeid.rpartition("/")[0].replace("/", ".")
+
+
+@pytest.hookimpl(trylast=True)
+def pytest_ddtrace_get_item_suite_name(item):
+    """
+    Extract suite name from a `pytest.Item` instance.
+    If the module path doesn't exist, the suite path will be reported in full.
+    """
+    # breakpoint()
```

Comment:
This should be removed because it lacks context and might be confusing to readers.

Reply:
Hah. Foiled.